waiter/docs/rest-api.md

```diff
@@ -7,6 +7,6 @@ This API description is meant to be an overview of methods available in the REST
 |`/apps/<service-id>`|`DELETE`|Kill all instances of a service. Note that outstanding/future requests may cause Waiter to restart instances of the service.|
 |`/service-id`       |`GET`   |Get the service-id for a given application. The service-id will be looked up using a combination of the headers/tokens provided in the request.|
 |`/status`           |`GET`   |Check on the status of Waiter.|
-|`/token`            |`GET`   |Retrieve information about a token. Token should be passed in the `X-Waiter-Token` header.|
-|`/token`            |`POST`  |Define a token. Tokens can store all or part of the service description.|
-|`/token`            |`DELETE`|Remove a token. Token should be passed in the `X-Waiter-Token` header.|
+|`/token`            |`GET`   |Retrieve information about a token. Token should be passed in the `X-Waiter-Token` header. An extra `include-deleted=true` query parameter can be used to retrieve a soft deleted token.|
+|`/token`            |`POST`  |Define a token. Tokens can store all or part of the service description. An extra `update-mode=admin` query parameter can be used to trigger a forced update of a token by a user with admin privileges.|
+|`/token`            |`DELETE`|Remove a token. Token should be passed in the `X-Waiter-Token` header. An extra `excise=true` query parameter can be used to trigger a hard delete of the token.|
```

Comment:
Agreed, I was thinking something along the lines of "Tokens can either be soft-deleted or hard-deleted. If a token is soft-deleted, it can still be retrieved by ... whereas hard-deleted tokens are permanently removed from the database ..."